controllers/articles.controller.js

```diff
@@ -43,6 +44,9 @@ exports.postCommentByArticleId = (req, res, next) => {
       .status(400)
       .send({ msg: "Username or body are required fields" });
   }
+  // let these undefined values go into the query, in which case
+  // PSQL would throw an error which can be handled.
+  // this approach also works^^
```

Comment:
Again, I'd advise getting rid of comments like this. A PR should represent a complete piece of work with all housekeeping on stuff like comments sorted and removed before submitting for a code review.